canonical-websites/build.snapcraft.io
# Updates npm-shrinkwrap

Seems that our npm-shrinkwrap.json could be out of date.

Running clean install (`rm -rf node_modules` and `npm i`) and recreating npm-shrinkwrap.json adds quite a bunch of stuff to this file which makes it quite confusing when trying to add new dependencies and seeing a lot of unrelated changes.

So this PR is an attempt to update npm-shrinkwrap.json without adding any new dependencies.

### QA
- checkout this branch
- rm -rf node_modules
- npm i
- run server, everything should work as expected
